tests/functional/olp-cpp-sdk-dataservice-read/CatalogClientTest.cpp

```diff
@@ -0,0 +1,598 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <chrono>
+#include <iostream>
+#include <regex>
+#include <string>
+#include <tuple>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <olp/authentication/AuthenticationCredentials.h>
+#include <olp/authentication/Settings.h>
+#include <olp/authentication/TokenEndpoint.h>
+#include <olp/authentication/TokenProvider.h>
+#include <olp/authentication/TokenResult.h>
+#include <olp/core/cache/DefaultCache.h>
+#include <olp/core/client/CancellationToken.h>
+#include <olp/core/client/HRN.h>
+#include <olp/core/client/HttpResponse.h>
+#include <olp/core/client/OlpClient.h>
+#include <olp/core/client/OlpClientFactory.h>
+#include <olp/core/client/OlpClientSettingsFactory.h>
+#include <olp/core/http/Network.h>
+#include <olp/core/http/NetworkRequest.h>
+#include <olp/core/http/NetworkResponse.h>
+#include <olp/core/logging/Log.h>
+#include <olp/core/porting/make_unique.h>
+#include <olp/core/utils/Dir.h>
+#include <olp/dataservice/read/PrefetchTilesRequest.h>
+#include <testutils/CustomParameters.hpp>
```

Comment:
In ApiTest.cpp you moved this include  (CustomParameters.hpp) in a separate group. So, better to stick with that approach